utility/src/main/java/org/oppia/util/parser/JsonParser.kt

```diff
@@ -0,0 +1,14 @@
+package org.oppia.util.parser
+
+import org.json.JSONArray
+
+/** This class parse json to mutable list of String. */
+class JsonParser {
+  fun parseJsonArrayToMutableListOfString(jsonArray: JSONArray): MutableList<String> {
```

Comment:
Let’s keep this in the other file rather than introducing a new class and method. 

Also, this can just return ``List``.

Finally, we can probably shorten the method name: parseJsonStringList.

Reply:
Resolved